assets/src/scss/base/_mixins.scss

```diff
@@ -205,3 +205,11 @@ $nested-comment-left-margin: 50px;
     }
   }
 }
+
+@mixin justifyContent($contentDistribution) {
```

Comment:
I've added this `mixin` to avoid duplicating code in all places that we're checking the browser support of the `justify-content` property. 

Here is more info https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

Reply:
We probably don't need to check supports anymore for this, the previous code was probably a leftover from when that used to be needed.

Reply:
https://caniuse.com/?search=flex-start

Reply:
Removed the support check about the `justify-content` property